### ISSUE TITLE
L. A. Times daily & weekly not downloading at all. 

### ISSUE THREAD

**Issue by marcodicalifornia** [kebernet/shortyz#180]
This has been going on for several months and I just figured out how to let you know.

**Comment by rmf2533** [comment]
I cannot download LA times puzzles.  I noticed this a few months ago, was not aware that this forum existed to identify such problems

**Comment by nelgin** [comment]
I'm having the same issue. I now can only get 3 puzzles per week. It looks like the author has given up so maybe line to look for a new app or something different to pass my time yet work my brain.

**Comment by rmf2533** [comment]
Agreed - I moved on to the Wall Street Journal crosswords.  Not an app, but
available through their website
